integrations/blaze/App.html

```diff
@@ -0,0 +1,13 @@
+<head>
+  <title>blaze-i18n-integration</title>
+</head>
+
+<body>
+  <h1>Example of i18n</h1>
+  {{> i18nTemplate}}
+</body>
+
+<template name="i18nTemplate">
+  <span>{{__ '.ok'}}</span>
```

Comment:
```suggestion
  <span>{{__ 'ok'}}</span>
```
Is this dot necessary?